### ISSUE TITLE
Java REST api `Period_identifier` enum is not public and cannot be accessed

### ISSUE THREAD

**Issue by ashkansk** [coinapi/coinapi-sdk#99]
I'm trying to use the java api and getting OHLCV historical data using the method `ohlcv_get_historical_timeseries` provided by the class `io.coinapi.rest.REST_methods` but the method needs the period of the time-series data to fetch. This argument is of the type `io.coinapi.rest.Period_identifier` which is not public and cannot be accessed from outside packages and results in compile errors if accessed and there's no way of calling this method!

I was wondering if I'm the first person who used this java api, since this code is committed around 10 months ago. Anyways is there something that I'm missing or this issue is really a problem?

P.S.
Apart from not sticking to Java naming conventions and some other structural problems of the code, the maven package with version 1.13 as mentioned in the documentation does not exist in maven central repository and I ended up using the version 1.2, however it seems it's the same code as the Git repo.

**Comment by Svisstack** [comment]
Hi @ashkansk,

You should not use the maven central repository package as we stopped publishing here and packages are available in the Github repository https://github.com/coinapi/coinapi-sdk/packages/397337

Please upgrade to the 1.14 that have a public on this enum.

**Comment by ashkansk** [comment]
Hi @Svisstack
Thank you for your quick response and quick fix. I was able to clone and build the code myself, but I'd rather use the package so I reverted my code again to use the latest package by adding Github repository as you mentioned, to my pom.xml but couldn't still download the package. I run into the below error and the package does not get installed.

 `[WARNING] The POM for io.coinapi.rest:v1:jar:1.14 is missing, no dependency information available`.
